### DISCUSSION TITLE
Pre-Configured Example Site

### DISCUSSION THREAD

**Discussion by pizzapanther** [requarks/wiki#6345] [Help / Questions]
I would like to demo wiki.js to my team as a recommendation. What would make this easy and successful is if there were some pre-existing data or a demo site that would show how a company would use wiki.js as an internal wiki. Is there any existing data fixtures I could load to jump start an install or demo site out there like this?

**Comment by Philipinho** [comment]
You can create a demo here https://hostwiki.com/demo.

**Comment by pizzapanther** [comment]
Great site, but I'm looking for something that has pre-loaded data I can show off, not just an empty wiki demo. And in particular something that shows off how a company might use the wiki internally.

**Comment by fnafox** [comment]
It might not be exactly what you're asking for, but why not clone [wiki-docs](https://github.com/requarks/wiki-docs) to your own instance? You might need to setup some basic permissions and configuration but at least the content is there.
